README.md

```diff
@@ -1,7 +1,13 @@
-﻿# [Group 4, FUML: Making UML Fun]
-
+# [FUML: Making UML Fun]
+ 
+ <img width="200" height="200" src="https://github.com/timjon/group4/blob/develop/Application/src/resources/logo.png">
```

Comment:
Whats this?

Reply:
Added logo to readme file

Reply:
Maybe it should say that then

Reply:
I can't add comments in the readme file, but if you view it on git you will see the logo

Reply:
Oh i see

Reply:
I'm confused, what changes were requested?

Reply:
@gusgolko confused about the readme file? tim just had a question and i answered. if you are confused about the pull request, then all that i have to change is the author tag if we agree on that in the meeting. other than that #12 is read to be merged